registry/client/errors.go

```diff
@@ -37,6 +38,22 @@ func (e *UnexpectedHTTPResponseError) Error() string {
 	return fmt.Sprintf("error parsing HTTP %d response body: %s: %q", e.StatusCode, e.ParseErr.Error(), string(e.Response))
 }
 
+// OAuthError is returned when the request could not be authorized
+// using the provided oauth token. This could represent a lack of
+// permission or invalid token given from a token server.
+// See https://tools.ietf.org/html/rfc6750#section-3
+type OAuthError struct {
+	ErrorCode   string
```

Comment:
Could you clarify that this will contain values from https://tools.ietf.org/html/rfc6750#section-3.1?